### DISCUSSION TITLE
[Idea] Image position per shop

### DISCUSSION THREAD

**Discussion by Dreimus** [PrestaShop/PrestaShop#35648] [Ideas]
### The problem

In image table, there is a position field which help each product image to be ordered by drag'n'drop in product edition.

If you are in a multi-shop context, there is no possibility to edit image positions differently between shops on the same product. The product image positions are common to all shops.

### The solution

migrate position field to image_shop and adapt the code to allow per shop image repositionning

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Good idea 👍🏻
